src/unix/internal.h

```diff
@@ -282,6 +282,7 @@ uv_handle_type uv__handle_type(int fd);
 FILE* uv__open_file(const char* path);
 int uv__getpwuid_r(uv_passwd_t* pwd);
 int uv__search_path(const char* prog, char* buf, size_t* buflen);
+void uv__chld(uv_signal_t* handle, int signum);
```

Comment:
edit this to take the uv_loop_t as the argument instead?

Reply:
done, split this into a signal handler (which must have this signature) and the function body that does the real work.